edit.html

```diff
@@ -33,10 +34,15 @@
 
     <script src="vendor/codemirror/addon/edit/matchbrackets.js"></script>
 
-    <link rel="stylesheet" href="vendor/codemirror/addon/lint/lint.css" />
+    <!-- CSSLint -->
```

Comment:
What if I used the "genericDisabledLabel" like that used in the Highlight dropdown?